pandas/tests/generic/test_generic.py

```diff
@@ -612,6 +612,16 @@ def test_pct_change(self, periods, fill_method, limit, exp):
         else:
             tm.assert_series_equal(res, Series(exp))
 
+    @pytest.mark.parametrize("DF,idx,first_idx,last_idx", [
```

Comment:
Some renaming suggestions for readability:
- `DF` --> `data`  (match `DataFrame` constructor)
- `idx` --> `index`  (match `DataFrame` constructor)
- `first_idx` --> `expected_first`  (follow standard expected/result unit test setup)
- `last_idx` --> `expected_last`  (follow standard expected/result unit test setup)

Reply:
Thanks - done